packages/flutter_tools/lib/src/device.dart

```diff
@@ -190,6 +190,12 @@ class DeviceManager {
           if (isDeviceSupportedForProject(device, flutterProject))
             device
       ];
+    } else if (devices.length == 1 &&
```

Comment:
What will this do if I'm in a flutter project that doesn't have desktop or web folders, but I have the options on in config?

Reply:
Both options?

Reply:
it should display "no supported devices for project"